stats_test.go

```diff
@@ -103,3 +104,84 @@ func TestNilAggregation(t *testing.T) {
 		t.Errorf("Expected: %v, Got: %v", fmt.Errorf("aggregation *view.Aggregation is not supported"), actual)
 	}
 }
+
+func Test_calculatePercentile(t *testing.T) {
+	var buckets []float64
+	for i := float64(-100); i < 100; i += 0.1 {
+		buckets = append(buckets, i)
+	}
+
+	normalDistribution := calculateNormalDistribution(buckets, 0, 100)
+	tsts := []struct {
+		description     string
+		expected        int64
+		percentile      float64
+		buckets         []float64
+		countsPerBucket []int64
+	}{
+		{
+			"Calculates 50th percentile for normal distribution",
```

Comment:
Should be enough to call these tests `50`, `75`, `99`, etc... Or even just doing an `t.Run(fmt.Sprintf("%v", tst.percentile), ...`. Even though the description is nice, it looks terrible in practice:
```
=== RUN   Test_calculatePercentile/Calculates_50th_percentile_for_normal_distribution
```
I'd say having this is better:
```
=== RUN   Test_calculatePercentile/50
```
It's also clearer. And easier to isolate in potential commands such as:
```
go test -run=Percentile/50
```

Reply:
👍